tests/test_api.py

```diff
@@ -30,139 +32,80 @@ def bliss_client(cwd, cfg):
+        monkeypatch.setenv("GITHUB_TOKEN", "test")
 
-    tractor_filename = "tractor-3366m010.fits"
-    cutout_filename = "cutout_336.635_-0.9600.fits"
-    cutout = download_git_lfs_file(
-        f"https://api.github.com/repos/prob-ml/bliss/contents/data/decals/{cutout_filename}",
-        headers={"Authorization": f"Bearer {os.environ['GITHUB_TOKEN']}"},
-    )
-    tractor = download_git_lfs_file(
-        f"https://api.github.com/repos/prob-ml/bliss/contents/data/decals/{tractor_filename}",
-        headers={"Authorization": f"Bearer {os.environ['GITHUB_TOKEN']}"},
-    )
-    cutout_path = Path(download_dir) / cutout_filename
-    tractor_path = Path(download_dir) / tractor_filename
-    cutout_path.write_bytes(cutout)
-    tractor_path.write_bytes(tractor)
-
-
-def download_rcf_from_git(run, camcol, field, cwd):
-    run_stripped = str(run).lstrip("0")
-    field_stripped = str(field).lstrip("0")
-    run6 = f"{int(run_stripped):06d}"
-    field4 = f"{int(field_stripped):04d}"
-
-    base_url = f"https://api.github.com/repos/prob-ml/bliss/contents/data/sdss/{run}/{camcol}"
-    save_path_rc = Path(cwd) / f"data/sdss/{run}/{camcol}"
-    save_path_rcf = save_path_rc / f"{field}"
-    save_path_rcf.mkdir(parents=True, exist_ok=True)
-
-    assert os.environ["GITHUB_TOKEN"] is not None, "GITHUB_TOKEN environment variable not found"
-    headers = {"Authorization": f"Bearer {os.environ['GITHUB_TOKEN']}"}
-    pf = download_git_lfs_file(f"{base_url}/photoField-{run6}-{camcol}.fits", headers)
-    po = download_git_lfs_file(
-        f"{base_url}/{field}/photoObj-{run6}-{camcol}-{field4}.fits", headers
-    )
-    psf = download_git_lfs_file(
-        f"{base_url}/{field}/psField-{run6}-{camcol}-{field4}.fits", headers
-    )
-    save_file(f"{save_path_rc}/photoField-{run6}-{camcol}.fits", pf)
-    save_file(f"{save_path_rc}/{field}/photoObj-{run6}-{camcol}-{field4}.fits", po)
-    save_file(f"{save_path_rc}/{field}/psField-{run6}-{camcol}-{field4}.fits", psf)
-
-    for band in "ugriz":
-        fpm = download_git_lfs_file(
-            f"{base_url}/{field}/fpM-{run6}-{band}{camcol}-{field4}.fits", headers
+        filename = "sdss_pretrained.pt"
+        bliss_client.load_pretrained_weights_for_survey(
+            survey="sdss",
+            filename=filename,
+            request_headers={"Authorization": f"Bearer {os.environ['GITHUB_TOKEN']}"},
         )
-        frame = download_git_lfs_file(
-            f"{base_url}/{field}/frame-{band}-{run6}-{camcol}-{field4}.fits", headers
+        not_found_err_msg = (
+            "pretrained weights "
+            + f"{bliss_client.cwd}/data/pretrained_models/{filename} "
+            + "not found"
+        )
+        assert Path(
+            bliss_client.cwd + f"/data/pretrained_models/{filename}"
+        ).exists(), not_found_err_msg
+
+    def test_train(self, bliss_client, monkeypatch):
+        monkeypatch.setattr(api, "_train", mock_train)
```

Comment:
Instead of overriding `bliss.api._train`, could we override `bliss.train` instead? And so we can import as `from bliss.api import BlissClient` too

Reply:
And this is good becos we don’t re-test bliss.train that already gets tested in tests like end_to_end

Reply:
It should be the same either way, since `bliss.api._train` is just an alias for `bliss.train.train` right? I just added a separate import for BlissClient so now it has both api and BlissClient in the global namespace.

Reply:
I don't think we should use an underscore import `_train`; like other underscore names it should be treated as private; just slightly cleaner to use 

```python
from bliss import train

monkeypatch.setattr(train, "train", mock_train)
```